packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -0,0 +1,244 @@
+import React, {
+  useState,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+import {
+  defaultPlaceholderValue,
+  isCurrentPlaceholder,
+  getDisplay,
+  getFirstValue,
+  getOptGroupKey,
+  geOptGroupKeyIndex,
+} from './NativeUtils';
+import {
+  optionPropType,
+  optGroupPropType,
+} from './NativePropTypes';
+
+import styles from './NativeSelect.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The string containing ids for elements to describe the select.
+   */
+  ariaDescribedBy: PropTypes.string,
+  /**
+   * The aria label string value for the select.
+   */
+  ariaLabel: PropTypes.string.isRequired,
+  /**
+   * Additional attributes to spread onto the select.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  attrs: PropTypes.object,
```

Comment:
Should we explain how `attrs` differs from custom props? spread on top level div vs spread on select component.

Reply:
Updated.